FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessaging.h

```diff
@@ -68,33 +68,37 @@ FOUNDATION_EXPORT const NSNotificationName FIRMessagingRegistrationTokenRefreshe
     NS_SWIFT_NAME(MessagingRegistrationTokenRefreshed);
 // clang-format on
 
+/**
+ * The domain used for all errors in Messaging.
+ */
+FOUNDATION_EXPORT NSString *const FIRMessagingErrorDomain NS_SWIFT_NAME(MessagingErrorDomain);
```

Comment:
kFIRMessagingDomain

Reply:
So then it's a breaking change only to be consistent with the style guide? I suppose that's ok since it's unlikely anyone is using this.

Reply:
Oh, sorry this isn't breaking it's a new public API. The previous API was internal only, so I changed it.

Reply:
Unfortunately that's not possible, it has to be a global constant. I'm in favour of still introducing it so we can take advantage of the better error handling sooner.